app/Repositories/LocationRepository.php

```diff
@@ -137,10 +137,11 @@ public function analystic(
                 'url' => route(
                     'calendar.location.detail_location', 
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Squiz.WhiteSpace.SuperfluousWhitespace.EndLine</br>**Severity**: error</br>**File**: app/Repositories/LocationRepository.php</br>Whitespace found at end of line